inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -0,0 +1,145 @@
+"""Wind tunnel scenario to run an object over an air flow."""
+
+from functools import singledispatchmethod
+import os
+import shutil
+from typing import Optional, List
+
+from absl import logging
+
+from inductiva.types import Path
+from inductiva.scenarios import Scenario
+from inductiva.simulation import Simulator
+from inductiva.fluids.simulators import OpenFOAM
+from inductiva.utils.templates import batch_replace_params_in_template
+from inductiva.utils.files import remove_files_with_tag
+
+OPENFOAM_TEMPLATE_INPUT_DIR = "openfoam_template"
+
+
+class WindTunnel(Scenario):
+    """Physical scenario of a general wind tunnel simulation."""
+
+    def __init__(self,
+                 object_path: str,
+                 flow_velocity: float = 50,
+                 domain: Optional[List[float]] = None):
+        """Initializes a `WindTunnel` object.
+        
+        Args:
+            object_path: Path to the object that is inserted in the wind tunnel.
+            flow_velocity: Velocity of the air flow in m/s.
+            domain: List containing the vertices describing the domain
+                of the wind tunnel in the [x, y, z] axes, in meters. This is a
+                natural description with the default OpenFOAM simulator.
+        """
+
+        self.object_path = object_path
+        self.flow_velocity = flow_velocity
+
+        if domain is None:
+            logging.info("Using a default domain: [[-5, 15], [-4, 4], [0, 8]].")
+            self.domain = [[-5, 15], [-4, 4], [0, 8]]
+
+    def simulate(self,
+                 simulator: Simulator = OpenFOAM(),
+                 output_dir: Optional[Path] = None,
+                 simulation_time: float = 100,
+                 write_interval: float = 50,
+                 n_cores: int = 1):
+        """Simulates the wind tunnel scenario.
+        
+        Args:
+            simulator: Simulator to use for the simulation.
+            output_dir: Path to the directory where the simulation output
+                is downloaded.
+            simulation_time: Simulation time, in seconds.
+            write_interval: Interval between simulation outputs, in seconds.
```

Comment:
We have called this `output_time_step` in previous scenarios. Rename it here for consistency?